### PR TITLE
Introduce GAPIC config for cloud language API.

### PR DESCRIPTION
**Pull request by jmuk**
- artman config: hand-written
- config yaml: hand-written
- gapic.yaml: generated from configgen
- modified to idempotent

### CODE REVIEW

File: google/cloud/language/v1beta1/language_gapic.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  python:
+    package_name: google.cloud.language.v1beta1
+  go:
+    package_name: google.golang.org/google/cloud/language/v1beta1
+  csharp:
+    package_name: Google.Cloud.Language.V1beta1
+  ruby:
+    package_name: Google::Cloud::Language::V1beta1
+  php:
+    package_name: Google\Cloud\Language\V1beta1
```
Comment (garrettjonesgoogle) [inline google/cloud/language/v1beta1/language_gapic.yaml]: capitalize the B here: V1Beta1

Comment (garrettjonesgoogle) [inline google/cloud/language/v1beta1/language_gapic.yaml]: To clarify: just for PHP.

Comment (jmuk) [inline google/cloud/language/v1beta1/language_gapic.yaml]: Done

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**